core/vision/src/main/java/com/arcrobotics/ftclib/vision/UGContourRingDetector.kt

```diff
@@ -14,6 +14,14 @@ class UGContourRingDetector(
         private val telemetry: Telemetry? = null,
         private var debug: Boolean = false,
 ) {
+    
+    companion object PipelineConfiguration {
+        var CAMERA_WIDTH = 320
+        var CAMERA_HEIGHT = 240
+
+        var HORIZON = 100
```

Comment:
Add a comment saying what this variable is maybe?

Reply:
It's pretty clear that it's a variable.

Reply:
But what it does is not clear.

Reply:
Do you mean explain what "horizon" is? I feel like that's something better covered in the docs. The reason for the previous comment was that it originally just returned height ZERO. Adding the comment there helps the user know what is integral to the detector (retrieving the stack height).

Reply:
If it can be covered in the docs, then great, but what the variable stands for may not be as intuitive as `CAMERA_HEIGHT` or `CAMERA_WIDTH`.

Reply:
FILTER_LOW_CUTOFF?

Reply:
Yeah that's true. It isn't named well. @JarnaChao09 perhaps name it CAMERA_VIEW_HORIZON? I do see it in the docs. There's a whole section on it.

Reply:
I can change it to CAMERA_VIEW_HORIZON, but that may lead to a breaking change since I would like UGContourPipeline follows the same naming scheme

Reply:
HORIZON_FILTER_CUTOFF

Reply:
ok, but that is still a breaking change

Reply:
I agree with HORIZON_FILER_CUTOFF. We do have to address the breaking change though.

Reply:
It's technically not a breaking change if I just change for PipelineConfiguration companion object, but I would like the Config companion object of the UGContourPipeline to follow the same variable names. However, I don't believe many teams are using the pipeline from the vision module. From the people on the discord, it looks like they are all just copying the code over and translating it back to Java if they don't use Kotlin

Reply:
There was that one guy, but I don't know if he understood how libraries worked. I'm sure people are using it as "inspiration", but I see what you mean

Reply:
I would still try to avoid breaking changes.

Reply:
This was before I added a comment above the variable for it. So seeing how there is now a comment about it, I don't think the variable name really means much, since if you want to know what the variable means, the comment is there explaining it.

Reply:
Well, that works for me.